bramkouwen/Neatly-In-Motion
# Creating Products Page

## What does this PR do?
* It creates products.html 
* It creates the page content for products.html

## How do you check this PR?
* Pull down the branch
* Check the added products.html file  
* Check the page content

## Screenshots 
None.
